tests/test_lag_2.py

```diff
@@ -92,6 +98,26 @@ def check_single_lag_lacp_rate(common_setup_teardown, nbrhosts, lag_name):
         if lag_rate_current_setting == 'fast':
             set_interface_lacp_rate(vm_host, neighbor_lag_intfs[0], 'normal')
 
+def check_single_lag(common_setup_teardown, nbrhosts, lag_name):
+    duthost, ptfhost, testbed, vm_neighbors, mg_facts, lag_facts, fanout_neighbors = common_setup_teardown
+    logging.info("Start checking single lap for: %s" % lag_name)
+
+    intf, po_interfaces = get_lag_intfs(lag_facts, lag_name)
+    po_flap = check_flap(lag_facts, lag_name)
+
+    # Figure out fanout switches info if exists for the lag member and run minlink test
+    if intf in fanout_neighbors.keys():
```

Comment:
There are no fanout switch in vstestbed, then I add this check to skip this test